content/docs/06-integrations/00-trident.md

```diff
@@ -0,0 +1,203 @@
+---
+title: 'Astra Trident'
+metaTitle: 'Astra Trident Integration with Palette'
+metaDescription: 'Astra Trident pack in Palette'
+hiddenFromNav: true
+type: "integration"
+category: ["storage"]
+logoUrl: ''
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Astra Trident
+
+Astra Trident, is an open-source project backed and maintained by NetApp and is designed to provide persistence storage to containerized applications using industry-standard interfaces such as the Container Storage Interface (CSI).
+
+Astra Trident deploys in Kubernetes clusters as pods and provides dynamic storage orchestration services for Kubernetes workloads. It enables containerized applications to quickly and easily consume persistent storage from NetApp’s portfolio that includes ONTAP (AFF/FAS/Select/Cloud/Amazon FSx for NetApp ONTAP), Element software (NetApp HCI/SolidFire), as well as the Azure NetApp Files service, and Cloud Volumes Service on Google Cloud.
+
+# Versions Supported
+
+<Tabs>
+
+<Tabs.TabPane tab="23.01.x" key="trident_23.01">
+
+## Prerequisites
+
+- All worker nodes in the Kubernetes cluster must be able to mount the volumes you have provisioned for your pods. To prepare the worker nodes, you must install NFS or iSCSI tools based on your driver selection. The docs on selecting the right tools, can be found [here](https://docs.netapp.com/us-en/trident/trident-use/worker-node-prep.html#selecting-the-right-tools).
+
+- Port requirements as listed in the table. 
+
+| Port  | Purpose                                                      |
+|-------|--------------------------------------------------------------|
+| 8443  | Backchannel HTTPS                                            |
+| 8001  | Prometheus metrics endpoint                                  |
+| 8000  | Trident REST server                                          |
+| 17546 | Liveness/readiness probe port used by Trident daemonset pods |
+
+<InfoBox>
+
+The liveness/readiness probe port can be changed during installation using the `--probe-port` flag. It is important to make sure this port isn’t being used by another process on the worker nodes. 
+
+</InfoBox>
+
+## Parameters
+
+| Parameter | Description |
```

Comment:
Let's add another column called "Default Values" and provide each default into its own respective cell. That makes it easier for the reader to scan the table.